### PR TITLE
Use tox-py in CI

### PR DESCRIPTION
**Pull request by adamchainz**
This is a [tox plugin I just made](https://pypi.org/project/tox-py/) to replace this shell-fu.

### CODE REVIEW

**Comment by adamchainz** [comment]
Old Pythons are killing this 👀

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -23,20 +23,22 @@ jobs:
...
-      run: python -m pip install --upgrade tox
-    - name: Run tox targets for ${{ matrix.python-version }}
       run: |
-        ENV_PREFIX=$(tr -C -d "0-9" <<< "${{ matrix.python-version }}")
-        TOXENV=$(tox --listenvs | grep "^py$ENV_PREFIX" | tr '\n' ',') python -m tox
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install --upgrade tox tox-py
+
+    - name: Run tox targets for ${{ matrix.python-version }}
+      run: tox --py current
```
Comment (charettes) [inline .github/workflows/main.yml:44]: I guess we could also use `tox-gh-actions`? That's what most Jazzband projects use


e.g.
https://github.com/charettes/django-seal/blob/abb29508fc6faca633541dcef00ef3fa9f1b684e/.github/workflows/test.yml#L36-L43
https://github.com/jazzband/django-model-utils/blob/master/.github/workflows/test.yml

Comment (adamchainz) [reply .github/workflows/main.yml:44]: Yeah if you want. I don't like having to redeclare which pyhons should run which envs, a bit redundant and another potential for failed maintenance. Given this package is getting obsolete I don't care that much about this PR though.

**Comment by charettes** [comment]
I appreciate your time trying to modernize the CI pipeline of this project but since it's pretty much deprecated I don't think it's worth getting this to work TBH.

**Comment by adamchainz** [comment]
Yeah same.

**Pull request closed without merging**